### PR TITLE
APM-698 Integrate NHS Login

### PR DESCRIPTION
**Pull request by artronics**
dummy change to start deployment

## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: tests/performance/README.md [hunk 1]
```diff
@@ -1,31 +0,0 @@
-# Performance Testing
```
Comment (Valentinolucas) [inline tests/performance/README.md:1]: Performance testing is used for load testing using locust.. Shall you keep this since is not related with your integration tests?

File: tests/performance/common/auth.py [hunk 2]
```diff
@@ -1,78 +0,0 @@
-import requests
```
Comment (Valentinolucas) [inline tests/performance/common/auth.py:1]: Performance testing is used for load testing using locust.. Shall you keep this since is not related with your integration tests?

File: tests/performance/locustfile.py [hunk 3]
```diff
@@ -1,29 +0,0 @@
-import os
```
Comment (Valentinolucas) [inline tests/performance/locustfile.py:1]: Performance testing is used for load testing using locust.. Shall you keep this since is not related with your integration tests?

File: tests/performance/README.md (hunk 1 above)
Comment (artronics) [reply tests/performance/README.md:1]: `performance` directory is back now

File: azure/templates/tests.yml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+      - ptl/app-credentials/jwt_testing/non-prod/ID_TOKEN_TESTING_PRIVATE_KEY
+      - ptl/app-credentials/jwt_testing/non-prod/ID_TOKEN_NHS_LOGIN_PRIVATE_KEY
+     config_ids: []
+     secret_ids: []
+
+  - bash: |
+     source ./test_env/bin/activate
+
+     pytest --service_name $(FULLY_QUALIFIED_SERVICE_NAME) --apigee_env $(APIGEE_ENVIRONMENT) \
+     --client_id eAMbFALgZFdnds2w5Z1u78AGCcJ0FtxQ \
```
[the remark this answers is not in the document]
Comment (artronics) [reply azure/templates/tests.yml:36]: `client_id` is not secret. I can put this as env var however, in the pipeline it should be hardcoded since, I know platforms doesn't store them as secrets. Because it'll be harcoded in the pipeline anyway I decided to hardcode it here as well to make life easier for devs.

**Review by Valentinolucas — APPROVED** [review]
